File: Iniciante/URI 1009.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val nome = leitor.next()
        val salarioFixo = leitor.nextDouble()
        val vendas = leitor.nextDouble()
        val total = ((vendas * 15) / 100) + salarioFixo
		printf("TOTAL = R$ %.2f\n", total)
    }
	
}

File: Iniciante/URI 1009.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val nome: String = leitor.next();
	val salarioFixo: Double = leitor.nextDouble();
	val vendas: Double = leitor.nextDouble();
	val total: Double = ((vendas * 15) / 100) + salarioFixo;
    
    println(String.format("TOTAL = R$ %.2f" , total));
}